src/components/ImageSocialShare.vue

```diff
@@ -8,6 +8,7 @@
     <social-share-buttons
       :shareURL="shareURL"
       :imageURL="imageURL"
+      :newprop="image_URL"
```

Comment:
Why did you call the new prop "newprop"? It's not descriptive of what the prop's value is. Please change "newprop" to a more descriptive name.

Reply:
You will need to update SocialShareButtons and the test spec for this component to reflect this change.